packages/app/src/pages/Rewards/Active/index.tsx

```diff
@@ -0,0 +1,220 @@
+// Copyright 2025 @polkadot-cloud/polkadot-staking-dashboard authors & contributors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import { planckToUnit } from '@w3ux/utils'
+import { useActiveAccounts } from 'contexts/ActiveAccounts'
+import { useBalances } from 'contexts/Balances'
+import { useNetwork } from 'contexts/Network'
+import { useActivePool } from 'contexts/Pools/ActivePool'
+import { useTokenPrices } from 'contexts/TokenPrice'
+import { AnimatePresence, motion } from 'framer-motion'
+import { useAverageRewardRate } from 'hooks/useAverageRewardRate'
+import { CardWrapper } from 'library/Card/Wrappers'
+import { Text } from 'library/StatCards/Text'
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { ButtonPrimary } from 'ui-buttons'
+import { CardHeader, PageRow, StatRow } from 'ui-core/base'
+import { RewardText, RewardsGrid } from '../Wrappers'
+
+export const Active = () => {
```

Comment:
Rename this component and its folder to `Overview`.

```suggestion
export const Overview = () => {
```